test/controllers/api/v2/content_credentials_controller_test.rb

```diff
@@ -44,7 +44,6 @@ def test_index_protected
 
     def test_content
       get :content, params: { :id => @gpg_key.id }
-      assert_response :success
```

Comment:
The body of this is just a string - not JSON. Since the tests validates the body I thought the simplest fix (so my new helper doesn't whine about it) was to just remove this assert